example_data/sampleFHIR_V2.json

```diff
@@ -881,9 +864,7 @@
           ]
         },
         "meta": {
-          "profile": [
-            "http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"
-          ]
+          "profile": ["http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/variant"]
```

Comment:
interesting that these are changing, I wonder if we should be running a precommit hook that formats the code in the same way